app/controllers/course/controller.rb

```diff
@@ -1,6 +1,8 @@
 class Course::Controller < ApplicationController
   load_and_authorize_resource :course
 
+  before_action :add_course_breadcrumb, only: [:show, :index, :edit]
```

Comment:
I personally think it's harmless to append it to all actions. There will be actions which are not restful (e.g. the course users controller)